package.json

```diff
@@ -122,6 +122,7 @@
     "react-addons-test-utils": "^16.0.0-alpha.3",
     "react-docgen-typescript-loader": "^2.1.0",
     "react-dom": "^16.5.0",
+    "react-powerplug": "^1.0.0",
```

Comment:
@ds300 - `react-powerplug` is indeed useful, but why not go with react upgrade for `<Toggle>` instead?

Reply:
Yeah I wanted to. We need to upgrade force and volt first since they consume reaction, right?

Reply:
Played around with this today. Turns out that storybooks story functions are not rendered as components but rather called as ordinary functions, so they don't support hooks anyway :cry: – and also after reading this https://github.com/reactjs/rfcs/pull/68#issuecomment-439314884 I'm not feeling too confident about using hooks in production code until the final release. So my motivation to upgrade react is flagging a bit.

Reply:
no prob! thanks for the RFC link 👍